BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -139,6 +141,32 @@ public ICollection<ITelemetry> TelemetryItems
             get; private set;
         }
 
+        /// <summary>
+        /// Gets the flush async id for the transmission.
+        /// </summary>
+        /// [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
was this meant to be non-browsable?

Reply:
Yes. We are marking this properties/methods as internal. I will get this attribute removed.